src/app/modules/event-types-out.ts

```diff
@@ -0,0 +1,19 @@
+// all events this components pushes for other components to use
+
+export const HTTP_CLIENT_NEEDED = 'httpClient.needed';
+
+export const NAVIGATION_DELETED = 'navigation.deleted';
+export const NAVIGATION_CREATED = 'navigation.created';
+export const NAVIGATION_RENAMED = 'navigation.renamed';
+export const NAVIGATION_OPEN = 'navigation.open';
+export const NAVIGATION_SELECT = 'navigation.select';
+
+export const WORKSPACE_OBSERVE = 'workspace.observe'; // ??
```

Comment:
Not needed anymore…? (As far as I can tell, it is only found in the workspace navigator, and only ever gets published in a spec file; so probably a feature we ended up never actually using…)